File: benchmarks/maps/bench.go
package main
import "fmt"
func map_test() int64 {
    m := make(map[int64]int64)
    var i int64
    for i = 0; i < 10000; i++ {
        m[i] = i * 2
    }
    var sum int64 = 0
    for i = 0; i < 10000; i++ {
        sum += m[i]
    }
    return sum
}
func main() {
    fmt.Println(map_test())
}

File: benchmarks/maps/Bench.java
import java.util.HashMap;
public class Bench {
    public static long map_test() {
        HashMap<Long, Long> map = new HashMap<>();
        for (long i = 0; i < 10000; i++) {
            map.put(i, i * 2);
        }
        long sum = 0;
        for (long i = 0; i < 10000; i++) {
            sum += map.get(i);
        }
        return sum;
    }
    public static void main(String[] args) {
        System.out.println(map_test());
    }
}
